docs/general-development/office-documentation-copyright-notice.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Office 2013 documentation copyright notice
+description: Provides a copyright notice for Office 2013 documentation.

```

Comment:
Too short per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main. I understand there isn't too much content in the file, but I think there's enough to get to 100 characters here.